antsibull/data/sphinx_init/conf_py.j2

```diff
@@ -0,0 +1,31 @@
+# This file only contains a selection of the most common options. For a full list see the
+# documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+project = 'Ansible collections'
+copyright = 'Ansible contributors'
+
+html_short_title = 'Ansible Collections Documentation'
```

Comment:
Maybe also add a normal title too?

Reply:
Improved in 2421c18cb71424fc43fc9b1f5cfb93666a2c9b9b.